README.md

```diff
@@ -26,22 +26,41 @@ Current status:
 Join us on Discord: https://discord.gg/tUFFk9Y
 
 
-## Getting started
+## Building jsparagus
+
+To build the parser by itself:
 
 ```sh
 make init
 make all
 ```
 
-(**Note:** This takes about 3 minutes to run on my laptop. Part of
-jsparagus is generated by a Python script, and the script is pretty
-slow. We're working on it!)
+The build takes about 3 minutes to run on my laptop.
 
 When it's done, you can:
 
 *   Run `make check` to make sure things are working.
 
-*   `cd crates/driver && cargo run` to test the JS parser and bytecode emitter.
+*   `cd crates/driver && cargo run -- -D` to try out the JS parser and bytecode emitter.
+
+
+## Building SpiderMonkey with jsparagus
+
+SpiderMonkey is the Mozilla JS engine. Currently it's possible to build
+SpiderMonkey with both the existing C++ parser and the jsparagus parser
+by configuring with `--enable-smoosh`. This builds with a specific
+known-good revision of jsparagus.
+
+To build SpiderMonkey with your own local jsparagus repo, you mainly
+just need to edit the file `js/src/frontend/smoosh/Cargo.toml` in your
+mozilla-central tree to point to your local repo instead of github.
+Then just make sure you've done `make all` in your jsparagus tree before
+running your SpiderMonkey build.
+
+If you don't like manually hacking `Cargo.toml` files,
+try @arai-a's [smoosh-tools](https://github.com/arai-a/smoosh-tools/).
+It will edit the file for you.
```

Comment:
maybe we should link the wiki here as well ? https://github.com/mozilla-spidermonkey/jsparagus/wiki/SpiderMonkey

Reply:
fixed!